src/metal.cs

```diff
@@ -5243,4 +5563,152 @@ interface MTLFunctionLogDebugLocation {
 		nuint Column { get; }
 	}
 
+	[Mac (12,0), iOS (15,0), TV (15,0), MacCatalyst (15, 0), NoWatch]
+	[BaseType (typeof (NSObject))]
+	interface MTLStitchedLibraryDescriptor : NSCopying
+	{
+		[Export ("functionGraphs", ArgumentSemantic.Copy)]
+		MTLFunctionStitchingGraph[] FunctionGraphs { get; set; }
+
+		[Export ("functions", ArgumentSemantic.Copy)]
+		IMTLFunction[] Functions { get; set; }
+	}
+
+	[Mac (12,0), iOS (15,0), NoTV, MacCatalyst (15, 0), NoWatch]
+	[BaseType (typeof (NSObject))]
+	interface MTLRenderPipelineFunctionsDescriptor : NSCopying
+	{
+		[NullAllowed, Export ("vertexAdditionalBinaryFunctions", ArgumentSemantic.Copy)]
+		IMTLFunction[] VertexAdditionalBinaryFunctions { get; set; }
+
+		[NullAllowed, Export ("fragmentAdditionalBinaryFunctions", ArgumentSemantic.Copy)]
+		IMTLFunction[] FragmentAdditionalBinaryFunctions { get; set; }
+
+		[NullAllowed, Export ("tileAdditionalBinaryFunctions", ArgumentSemantic.Copy)]
+		IMTLFunction[] TileAdditionalBinaryFunctions { get; set; }
+	}
+
+	[Mac (12, 0), iOS (15, 0), NoTV, MacCatalyst (15, 0), NoWatch]
+	[BaseType (typeof (NSObject))]
+	interface MTLMotionKeyframeData {
```

Comment:
```suggestion
	interface MtlMotionKeyframeData {
```
Not sure, but wanted to check this?